lib/WebDriver/Session.php

```diff
@@ -476,7 +476,13 @@ private function serializeArguments(array $arguments)
         foreach ($arguments as $key => $value) {
             // Potential compat-buster, i.e., W3C-specific
             if ($value instanceof Element) {
-                $arguments[$key] = [Container::WEBDRIVER_ELEMENT_ID => $value->getID()];
+                // preferably we want to detect W3C support and never set nor parse
```

Comment:
is W3C support somehow reported & nicely detectable?

Reply:
There was a w3c flag at one point but only on `master`. I'll need to revisit.